package.json

```diff
@@ -45,5 +45,7 @@
     "coverageDirectory": "./coverage/",
     "collectCoverage": true
   },
-  "dependencies": {}
```

Comment:
This dependency must be added inside `packages/inquirer/package.json` (that's because this project is a monorepo)

Reply:
thanks. i also added it to packages/core/package.json